packages/react-router-dom/modules/Router.js

```diff
@@ -1 +1 @@
-export { Router as default } from 'react-router'
+export { default } from 'react-router/Router'
```

Comment:
Actually to improve on this, my initial idea was a bit of overkill.

I was thinking of a custom Babel plugin that will rewrite the export to right direct import depending on the build. I personally feel that the gain will be very small but I am available to do this just for the fun of it.

@timdorr what do you think? Custom babel plugin seems too much?

So I decided to go the lazy route. Check it out.